python/lsst/sims/photUtils/Photometry.py

```diff
@@ -569,7 +569,8 @@ def sum_magnitudes(self, disk = None, bulge = None, agn = None):
             raise RuntimeError("Arguments of sum_magnitudes need to be " +
                                "either floats or numpy arrays; you appear to have passed %s " % baselineType)
 
-        mm_o = 22.
+        mm_0 = 22.
```

Comment:
I'm actually not sure why it is 22 (and, in fact, it is not 22 in the Sed.calcMag methods).  For this purpose, however, it should not matter, right?  I'm converting magnitudes into fluxes, renormalizing the fluxes by 10^(-0.4*mm_0), summing the fluxes, and then dividing out the normalizing factor when I convert back into magnitudes.